.travis.yml

```diff
@@ -4,4 +4,4 @@ jdk:
 - openjdk7
 
 script:
-- mvn clean deploy
+- mvn clean test
```

Comment:
Good point. Sounds like this will be a trickier refactor than I thought, considering we'd have to update Travis, Jenkins, Maven, and potentially refactor some of our tests. Moving this to JIRA https://sagebionetworks.jira.com/browse/BRIDGE-781